core/src/main/scala/besom/internal/codecs.scala

```diff
@@ -229,6 +229,12 @@ object Decoder extends DecoderInstancesLowPrio1:
       if v.kind.isBoolValue then v.getBoolValue.valid
       else error(s"$label: Expected a boolean, got: '${v.kind}'", label).invalid
 
+  given outputDecoder[A](using inner: Decoder[A]): Decoder[Output[A]] = new Decoder[Output[A]]:
+    override def decode(value: Value, label: Label)(using Context): ValidatedResult[DecodingError, OutputData[Output[A]]] =
+      inner.decode(value, label).map(_.map(Output(_)))
```

Comment:
This is wrong - it misses the whole point of this change! This works in happy case only and we're not interested in happy case because if anything goes wrong with decoding on this level with this implementation it will crash deserialization of the whole entity instead of this single field like we intended. You should create a redeem method on `ValidatedResult` like this:
```scala
    def redeem[B](fe: NonEmptyVector[E] => B, fa: A => B): ValidatedResult[E, B] =
      result.map {
        case Valid(a) => Valid(fa(a))
        case Invalid(e) => Valid(fe(e))
      }
```
and then handle the invalid case by producing a failed `Output` with `Output.fail` containing an `AggregatedDecodingError`. For happy `Valid` path just wrap in `Output()`.